### DISCUSSION TITLE
Time taken to index the code pushed to github repository

### DISCUSSION THREAD

**Discussion by AmruthGnanasekaran** [community/community#129971] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have pushed code yesterday in the one of the repo. But I cannot able to find out using the search/code api. How long it will take to index the code in repository ?

**Comment by look** [comment, marked as answer]
It depends on how big your repository is, but typically changes are available within a few minutes. If your repository is not indexed after an extended period of time, you can file a support ticket.
